scripts/build/testing.mk

```diff
@@ -285,6 +221,149 @@ start-ethereumjs:
 	--rpc \
 	--rpcAddr 0.0.0.0
 
+#################
+#    Bepolia    #
+#################
+
+TESTNET_CHAIN_SPEC = testnet
+BEPOLIA_NETWORK_FILES_DIR = ${TESTAPP_FILES_DIR}/../networks/80069
+BEPOLIA_ETH_GENESIS_PATH = ${BEPOLIA_NETWORK_FILES_DIR}/eth-genesis.json
+
+start-bepolia:
+	@JWT_SECRET_PATH=$(JWT_PATH) \
+	CHAIN_SPEC=$(TESTNET_CHAIN_SPEC) \
+	${TESTAPP_FILES_DIR}/entrypoint.sh
+
+# NOTE: Peers are used as bootnodes for the Geth node.
+start-geth-bepolia:
+	# TODO: Update to use latest Geth once ready
+	$(call ask_reset_dir_func, $(ETH_DATA_DIR))
+	docker run \
+	--rm -v $(PWD)/${TESTAPP_FILES_DIR}:/${TESTAPP_FILES_DIR} \
+	--rm -v $(PWD)/${BEPOLIA_NETWORK_FILES_DIR}:/${BEPOLIA_NETWORK_FILES_DIR} \
+	-v $(PWD)/.tmp:/.tmp \
+	ethereum/client-go:v1.14.13 init \
+	--datadir ${ETH_DATA_DIR} \
+	${BEPOLIA_ETH_GENESIS_PATH}
+
+	@# Read bootnodes from the file; the file is mounted into the container.
+	@bootnodes=`cat $(PWD)/$(BEPOLIA_NETWORK_FILES_DIR)/el-peers.txt`; \
+	echo "Using bootnodes: $$bootnodes"; \
+	docker run \
+	-p 30303:30303 \
+	-p 8545:8545 \
+	-p 8551:8551 \
+	--rm -v $(PWD)/${TESTAPP_FILES_DIR}:/${TESTAPP_FILES_DIR} \
+	--rm -v $(PWD)/${BEPOLIA_NETWORK_FILES_DIR}:/${BEPOLIA_NETWORK_FILES_DIR} \
+	-v $(PWD)/.tmp:/.tmp \
+	ethereum/client-go:v1.14.13 \
+	--http \
+	--http.addr 0.0.0.0 \
+	--http.api eth,net \
+	--authrpc.addr 0.0.0.0 \
+	--authrpc.jwtsecret $(JWT_PATH) \
+	--authrpc.vhosts "*" \
+	--datadir ${ETH_DATA_DIR} \
+	--ipcpath ${IPC_PATH} \
+	--syncmode=full \
+	--bootnodes $$bootnodes
+
+start-reth-bepolia:
+	$(call ask_reset_dir_func, $(ETH_DATA_DIR))
+	@trustedpeers=`cat $(PWD)/$(BEPOLIA_NETWORK_FILES_DIR)/el-peers.txt`; \
+	echo "Using truted peers: $$trustedpeers"; \
+	docker run \
+	-p 30303:30303 \
+	-p 8545:8545 \
+	-p 8551:8551 \
+	--rm -v $(PWD)/${TESTAPP_FILES_DIR}:/${TESTAPP_FILES_DIR} \
+	--rm -v $(PWD)/${BEPOLIA_NETWORK_FILES_DIR}:/${BEPOLIA_NETWORK_FILES_DIR} \
+	-v $(PWD)/.tmp:/.tmp \
+	ghcr.io/paradigmxyz/reth node \
+	--chain ${BEPOLIA_ETH_GENESIS_PATH} \
+	--http \
+	--http.addr "0.0.0.0" \
+	--http.api eth,net \
+	--authrpc.addr "0.0.0.0" \
+	--authrpc.jwtsecret $(JWT_PATH) \
+	--datadir ${ETH_DATA_DIR} \
+	--ipcpath ${IPC_PATH} \
+	--trusted-peers $$trustedpeers
+
+#################
+#    Mainnet    #
+#################
+
+MAINNET_CHAIN_SPEC = mainnet
+MAINNET_NETWORK_FILES_DIR = ${TESTAPP_FILES_DIR}/../networks/80094
+MAINNET_ETH_GENESIS_PATH = ${MAINNET_NETWORK_FILES_DIR}/eth-genesis.json
+
+start-mainnet:
+	@JWT_SECRET_PATH=$(JWT_PATH) \
+	CHAIN_SPEC=$(MAINNET_CHAIN_SPEC) \
+	${TESTAPP_FILES_DIR}/entrypoint.sh
+
+# NOTE: By default this will use the EL peers as your bootnodes. If you want specific 
+# discovery bootnodes by region, refer to testing/networks/80094/el-bootnodes.txt
```

Comment:
ideally, user uses bootnodes specifically only if they need to for issues with bad discovery, etc.